README.md

```
@@ -1,28 +1,237 @@
+
+### Generate Auth Secret:
+
+-   Run the following command:
+
+```bash
+npx auth secret
+```
+
+> This will generate a `.env.local` file with `AUTH_SECRET` environment variable in it.
+
+### Set Environment variables `.env.local`.
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '.'

(MD026, no-trailing-punctuation)